src/jsep.js

```diff
@@ -468,6 +445,13 @@ export class Jsep {
 		let ch, to_check, tc_len, node;
 
 		this.gobbleSpaces();
+		this.node = null;
+		Jsep.hooks.run('gobble-token', this);
+		if (this.node) {
+			Jsep.hooks.run('after-token', this);
```

Comment:
Do we actually need two successive hooks here?

Reply:
Just for consistency to run before all exit points of the method. I could potentially limit to just the ones needed for the plugins that we'll initially write? And I might be able to make them use one of the other hooks, but I was using the 'after'token' for objects, template literals, and the 'new' keyword.

Reply:
Ok, that makes sense.